### ISSUE TITLE
update failed for AnAction with ID=RunClass: Argument for @NotNull parameter 'module' of com/intellij/openapi/roots/ModuleRootManager.getInstance must not be null

### ISSUE THREAD

**Issue by CreamyCookie** [raniejade/spek-idea-plugin#54]
This error message comes up randomly. (I didn't do anything Spek-related.)

```
update failed for AnAction with ID=RunClass: Argument for @NotNull parameter 'module' of com/intellij/openapi/roots/ModuleRootManager.getInstance must not be null
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'module' of com/intellij/openapi/roots/ModuleRootManager.getInstance must not be null
	at com.intellij.openapi.roots.ModuleRootManager.getInstance(ModuleRootManager.java)
	at org.jetbrains.spek.idea.SpekRunConfigurationProducer.isConfigurationFromContext(SpekRunConfigurationProducer.kt:124)
	at org.jetbrains.spek.idea.SpekRunConfigurationProducer.isConfigurationFromContext(SpekRunConfigurationProducer.kt:25)
	at com.intellij.execution.actions.RunConfigurationProducer.findExistingConfiguration(RunConfigurationProducer.java:210)
	at com.intellij.execution.actions.ConfigurationContext.findExisting(ConfigurationContext.java:229)
	at com.intellij.execution.actions.BaseRunConfigurationAction.update(BaseRunConfigurationAction.java:184)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:132)
	at com.intellij.openapi.actionSystem.impl.Utils.doUpdate(Utils.java:235)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:180)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:205)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:205)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:205)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:205)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:100)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:341)
	at com.intellij.openapi.actionSystem.impl.ActionPopupMenuImpl$MyMenu.show(ActionPopupMenuImpl.java:98)
	at com.intellij.ide.ui.customization.CustomizationUtil$2.invokePopup(CustomizationUtil.java:286)
	at com.intellij.ui.PopupHandler.mousePressed(PopupHandler.java:48)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:280)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.Component.processMouseEvent(Component.java:6538)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at com.intellij.ui.treeStructure.Tree.processMouseEvent(Tree.java:429)
	at com.intellij.ide.dnd.aware.DnDAwareTree.processMouseEvent(DnDAwareTree.java:58)
	at java.awt.Component.processEvent(Component.java:6306)
	at java.awt.Container.processEvent(Container.java:2237)
	at java.awt.Component.dispatchEventImpl(Component.java:4897)
	at java.awt.Container.dispatchEventImpl(Container.java:2295)
	at java.awt.Component.dispatchEvent(Component.java:4719)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4889)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4523)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4467)
	at java.awt.Container.dispatchEventImpl(Container.java:2281)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4719)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.awt.EventQueue$4.run(EventQueue.java:735)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:821)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:645)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

spek version: 1.1.4
plugin version: 0.43
IDEA version: 2017.2.5

**Comment by raniejade** [comment]
Not a problem with spek based on the stacktrace
```
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'module' of com/intellij/openapi/roots/ModuleRootManager.getInstance must not be null
	at com.intellij.openapi.roots.ModuleRootManager.getInstance(ModuleRootManager.java)
```

`ModuleRootManager.getInstance` returned null which it shouldn't suppose to.

**Comment by CreamyCookie** [comment]
@raniejade So, should I report it on YouTrack (Jetbrains issue tracker)?

**Comment by raniejade** [comment]
On second thought, it looks like an issue with the plugin. Let me look into it some more.

**Comment by raniejade** [comment]
Fixed, but it's not part of `0.5.3` release.

**Comment by CreamyCookie** [comment]
Cool, thanks! :)
